app/src/main/java/org/mozilla/focus/session/SessionManager.java

```diff
@@ -71,14 +71,11 @@ private void createSessionFromIntent(Context context, SafeIntent intent) {
         final String action = intent.getAction();
 
         if (Intent.ACTION_VIEW.equals(action)) {
-            final Source source;
             if (intent.hasExtra(HomeScreen.ADD_TO_HOMESCREEN_TAG)) {
-                source = Source.HOME_SCREEN;
-                boolean blockingEnabled = intent.getBooleanExtra(HomeScreen.BLOCKING_ENABLED, true);
-                createSession(context, source, intent, intent.getDataString(), blockingEnabled);
+                final boolean blockingEnabled = intent.getBooleanExtra(HomeScreen.BLOCKING_ENABLED, true);
+                createSession(context, Source.HOME_SCREEN;, intent, intent.getDataString(), blockingEnabled);
```

Comment:
nit: not great for bisection that we break the build in this commit (with the semicolon) but revert it in a later commit. That being said, if we create a no-ff merge commit, I assume it wouldn't go into branches when bisecting. Also, I don't know how often bisecting is actually used when we can just bisect Nightly builds.